docs/troubleshooting/cannot-add-existing-policy.mdx

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: Existing policy can't be added
+---
+
+## Existing policy can't be added
```

Comment:
This needs to be an H1 (#) for the title to render correctly (instead of the name of the file).

Also, let's be 100% consistent - the title of the page needs to be a single # (h1) and then subsequent headings need to be ##, ###, etc